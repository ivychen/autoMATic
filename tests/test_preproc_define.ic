/* autoMATic Compiler - Spring 2018 - COMS4115 PLT
 by Ivy Chen ic2389, Nguyen Chi Dung ncd2118,
 Nelson Gomez ng2573, Jimmy O'Donnell jo2474 */

#include "lib/stdlib_test.ic"

#define AYYLMAO 69
#define MORE AYYLMAO

int main() {
	println(number());
	println(AYYLMAO);
	println(MORE);
	return 0;
}
