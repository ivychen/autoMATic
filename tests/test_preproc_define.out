100
69
69
